mpc/share-conversion/mpc-share-conversion/src/tape.rs

```diff
@@ -0,0 +1,95 @@
+use crate::TapeVerificationError;
+
+use mpc_share_conversion_core::{fields::Field, Share};
+use rand::SeedableRng;
+use rand_chacha::ChaCha12Rng;
+
+/// A tape which allows to record inputs and outputs of the conversion.
+/// The sender can reveal his tape (thus revealing all his secret inputs) to the receiver
+/// who will check it against her tape to detect any malicious behaviour of the sender.
```

Comment:
Can we make an extra note here that rng seed is also revealed by the sender, otherwise it may seems as if tape is all that is needed for the replay.